source/fe/fe_system.cc

```diff
@@ -1208,10 +1208,6 @@ FESystem<dim, spacedim>::compute_fill(
          ExcInternalError());
   const InternalData &fe_data = static_cast<const InternalData &>(fe_internal);
 
-  // Either dim_1==dim
-  // (fill_fe_values) or dim_1==dim-1
-  // (fill_fe_(sub)face_values)
-  Assert(dim_1 == dim || dim_1 == dim - 1, ExcInternalError());
```

Comment:
Your other changes shouldn't affect this. Why are you removing it?

Reply:
There is no more `dim_1` template so it has to go.

Reply:
Ah, doh!